Wflow/test/run_sbm_gwf.jl

```diff
@@ -110,15 +111,20 @@ Wflow.close_files(model; delete_output = false)
 
 # test local-inertial option for river and overland flow routing
 tomlpath = joinpath(@__DIR__, "sbm_gwf_config.toml")
-config = Wflow.Config(tomlpath)
-config.model.river_routing = "local-inertial"
-config.model.land_routing = "local-inertial"
-
-config.input.static.river_bank_water__elevation = "bankfull_elevation"
-config.input.static.river_bank_water__depth = "bankfull_depth"
-config.input.static.land_surface_water_flow__ground_elevation = "wflow_dem"
-
-pop!(Dict(config.state.variables), "land_surface_water__instantaneous_volume_flow_rate")
+config = Wflow.Config(
+    tomlpath;
+    model_river_routing = "local-inertial",
+    model_land_routing = "local-inertial",
+)
+
+config.input.static.dict["river_bank_water__elevation"] =
+    Wflow.InputEntry(; standard_name = "bankfull_elevation")
+config.input.static.dict["river_bank_water__depth"] =
+    Wflow.InputEntry(; standard_name = "bankfull_depth")
+config.input.static.dict["land_surface_water_flow__ground_elevation"] =
+    Wflow.InputEntry(; standard_name = "wflow_dem")
```

Comment:
Related to my earlier comment, here I am also doubting whether this new implementation is actually easier. To me, the previous option is much easier and more straightforward. Similar as before, only expert users will touch this interface, but still a remark I wanted to make